marlowe/marlowe.cabal

```diff
@@ -63,31 +59,31 @@ library
     ghc-options: -fplugin-opt Language.PlutusTx.Plugin:defer-errors
   build-tool-depends: unlit:unlit -any
 
-test-suite marlowe-test
-    default-language: Haskell2010
-    hs-source-dirs: test
-    type: exitcode-stdio-1.0
-    main-is: Spec.hs
-    other-modules:
-      Spec.Common
-      Spec.Marlowe
-      Spec.Actus
-    build-depends:
-        base >=4.9 && <5,
-        containers -any,
-        hedgehog -any,
-        memory -any,
-        bytestring -any,
-        tasty -any,
-        tasty-hunit -any,
-        tasty-hedgehog >=0.2.0.0,
-        text -any,
-        plutus-wallet-api -any,
-        plutus-emulator -any,
-        marlowe,
-        plutus-tx -any,
-        template-haskell -any
-  ghc-options:
-    -Wall -Wnoncanonical-monad-instances
-    -Wincomplete-uni-patterns -Wincomplete-record-updates
-    -Wredundant-constraints -Widentities
+-- test-suite marlowe-test
```

Comment:
Probably shouldn't be commented out.

Reply:
@nau will have some new tests at some point, these tests are no longer relevant so I have removed everything.